testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -291,6 +309,44 @@ def test_write_single_frame_AtomGroup(self, universe2, outfile):
                                                "agree with original coordinates from frame %d" %
                                                u.trajectory.frame)
 
+    def test_write_nodims(self, universe_and_expected_dims, outfile):
+        """
+        Test :code:`PDBWriter` for universe without cell dimensions.
+
+        Notes
+        -----
+        Test fix for Issue #2679.
+        """
+
+        u, expected_dims = universe_and_expected_dims
+
+        if expected_dims is None:
+            assert u.dimensions is None
+        else:
+            assert np.allclose(u.dimensions, expected_dims)
+
+        expected_msg = "Unit cell dimensions not found. CRYST1 record set to unitary values."
+
+        with pytest.warns(UserWarning, match=expected_msg):
+
+            u.atoms.write(outfile)
+
+        uout = mda.Universe(outfile)
+
```

Comment:
Ideally at this point the reader should also recognise this as a default box, could you check either that the dimensions returned are [0, 0, 0, 0, 0, 0] or that you get raise the "1 A^3 CRYST1 record" warning?